netmap/netmap_contract.go

```diff
@@ -601,6 +602,23 @@ func ListConfig() []record {
 	return config
 }
 
+// RegisterNewEpochRecipient registers passed contract as a NewEpoch event
+// receiver. Such a contract must have a `NewEpoch` method with a single numeric
+// argument. Transactions that call RegisterNewEpochRecipient must be witnessed
+// by the Alphabet.
+// Produces `NewEpochHookRegistration` notification event with a just registered
+// recipient in a success case.
+func RegisterNewEpochRecipient(contract interop.Hash160) {
+	common.CheckAlphabetWitness(common.AlphabetAddress())
+
+	ctx := storage.GetContext()
+	key := append([]byte(newEpochRecipientsKey), contract...)
+
+	storage.Put(ctx, key, 1)
```

Comment:
Put an empty byte array, save a bit of storage!

Reply:
hm, did not know it was possible to store nothing, fixed